src/foundry/client/pixi/webgl/shaders/fragment-channel-mixin.d.mts

```diff
@@ -0,0 +1,42 @@
+import type { Mixin } from "../../../../../types/utils.d.mts";
+
+export {};
+
+/** @remarks Class name adjusted to avoid name collision with function */
+declare class AdaptiveFragmentChannelMixinClass {
+  /** @privateRemarks All mixin classses should accept anything for its constructor. */
+  constructor(...args: any[]);
+
+  /**
+   * The fragment shader which renders this filter.
+   * A subclass of AdaptiveFragmentChannelMixin must implement the fragmentShader static field.
+   * @defaultValue `null`
+   */
+  static adaptiveFragmentShader: ((channel: AdaptiveFragmentChannel.Channel) => string) | null;
+
+  /**
+   * A factory method for creating the filter using its defined default values
+   * @param options - Options which affect filter construction
+   */
+  static create(options?: {
+    /**
+     * Initial uniforms provided to the filter
+     */
+    uniforms?: AbstractBaseShader.Uniforms;
+
+    /**
+     * A color channel to target for masking.
+     * @defaultValue `r`
+     */
+    channel?: string;
+  }): InverseOcclusionMaskFilter;
```

Comment:
`InexactPartial`?

Reply:
I don't know exactly what you mean by that, unfortunately. 

But, this section has an additional problem: `InverseOcclusionMaskFilter` doesn't exist anymore (but I only deleted it *after* working on this file, so I didn't notice the error previously).

Foundry currently has the return type as `PIXI.Shader | PIXI.Filter` but I'm unsure, given other notes on this PR, if simply copying that is appropriate here.